.opspec/release/to-github/op.yml

```diff
@@ -28,31 +33,18 @@ run:
           dotGitDir: $(../../../.git)
         outputs:
           commit:
-    - container:
-        image:
-          ref: alpine
-        cmd:
-          - sh
-          - -c
-          - |
-            if echo $(version) | grep -q -E '[0-9]+.[0-9]+.[0-9]+-[A-Za-z0-9]{1,}'; then
-              echo -n true > /prerelease
-            else
-              echo -n false > /prerelease
-            fi
-        files:
-          /prerelease: $(prerelease)
     - op:
         ref: github.com/opspec-pkgs/github.release.create#2.0.0
         inputs:
           commitish: $(commit)
+          description: $(latestRelease.description)
```

Comment:
use changelog details as release description so user facing product changes are presented as opposed to GitHub default of summary of commits (which give many irrelevant non-product facing changes).